credmaster.py

```diff
@@ -589,33 +582,18 @@ def log_entry(entry):
 
 	lock.acquire()
 
-	ts = datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3]
-	print('[{}] {}'.format(ts, entry))
+	ts = utc_now.strftime('%Y-%m-%d %H:%M:%S.%f')[:-3]
+	print(f"[{ts}] {entry}")
 
 	if outfile is not None:
-		with open(outfile + "-credmaster.txt", 'a+') as file:
-			file.write('[{}] {}'.format(ts, entry))
-			file.write('\n')
+		with open(outfile, 'a+') as file:
```

Comment:
Add back in the "-credmaster.txt" addition to the outfile name, used to distinguish between standard outfile and valid user outfile